commons/commons-model-extended/src/main/kotlin/ebi/ac/uk/extended/model/ExtendedModel.kt

```diff
@@ -68,11 +73,18 @@ data class ExtSubmission(
     val releaseTime: OffsetDateTime?,
     val modificationTime: OffsetDateTime,
     val creationTime: OffsetDateTime,
+    val section: ExtSection,
     val attributes: List<ExtAttribute> = listOf(),
     val tags: List<ExtTag> = listOf(),
-    val accessTags: List<ExtAccessTag> = listOf(),
-    val section: ExtSection
-)
+    val projects: List<Project> = listOf(),
+    val stats: List<ExtStat> = listOf()
+) {
+    // TODO: add custom serializer/deserializer to avoid json annotation in model
+    @get:JsonProperty(access = READ_ONLY)
+    val accessTags: List<ExtAccessTag> by AccessTagDelegate()
```

Comment:
I'm not so sure about having both, AccessTags and Projects just to be able to add "Public". Do we actually need/have any other non-project related access tag besides "Public"?

I'm asking this because we'll have two almost duplicated lists in the model. Maybe we could just add "Public" to projects or just leave AcessTags as it's now.

Reply:
right now it does not make much sense as both concept has been merged, submission project and submission access tag but actually there are different. 

Adding Public to protect implly that latter we need to removed or ignore as "Public" is not a project. The idea is that access tag are removed from the database and project relation is added to submission.